x-pack/plugin/security/qa/multi-cluster/build.gradle

```diff
@@ -28,7 +28,7 @@ tasks.named("javaRestTest") {
   exclude '**/RemoteClusterSecurityBwcRestIT.class'
 }
 
-BuildParams.bwcVersions.withWireCompatible(v -> v.before('8.8.0')) { bwcVersion, baseName ->
+BuildParams.bwcVersions.withWireCompatible(v -> v.before(BuildParams.isSnapshotBuild() ? '8.8.0' : '8.9.1')) { bwcVersion, baseName ->
```

Comment:
This needs to be `8.9.1` for now because `8.9.1` (unreleased) is a snapshot build even in release CI. It is not really a valid scenario, i.e. upgrading from snapshot to a release version. So skipping it is reasonable. It would be better if we could detect this instead of hard-coding, i.e. once 8.9.1 is released, it should allow test to run for upgrading from 8.9.1. I asked the question in the es-delivery channel. For now, hard-coding it should be fine.